lib/api.ts

```diff
@@ -180,6 +181,20 @@ export async function getContributorBySlug(file: string, detail = false) {
 
   const calendarData = getCalendarData(weightedActivity.activity);
 
+  const isNewContributor = (() => {
+    const { activity } = activityData;
+
+    const aWeekAgo = sub(new Date(), { days: 7 });
+
+    const firstActivity = activity
+      .filter((act) => act.type === "pr_merged")
+      .sort(
+        (a, b) => new Date(a.time).getTime() - new Date(b.time).getTime(),
+      )[0];
```

Comment:
```suggestion
      .sort(compareAsc)[0];
```

Reply:
Date-fns seems pretty interesting to me, but I don't know much about it yet 😂.